components/accept-terms.jsx

```diff
@@ -55,7 +53,7 @@ export function AcceptTerms({
 		...(isChecked && { defaultChecked: true }),
 	};
 
-	const authFirstStepTerms = (
+	const terms = (
 		<div className="auth-first-step-terms">
```

Comment:
this class name should be changed to something more descriptive of the feature bloc.
```suggestion
		<div className="privacy-policy-terms">
```
?